server/views/macros/adaTable.njk

```diff
@@ -15,86 +15,98 @@ totalText: optional,  defaults to 'Total' , the text displayed next to the total
-                  <th class="govuk-table__header govuk-!-width-one-quarter" scope="col">Heard at</th>
+                  <td class="govuk-table__cell">
+                    {{ charge.chargeNumber }}
+                  </td>
+                  <td class="govuk-table__cell">
+                    {{ charge.toBeServed }}
+                  </td>
+                  <td class="govuk-table__cell">
+                    {{ charge.heardAt }}
+                  </td>
                   {% if not params.hideStatuses %}
-                    <th class="govuk-table__header" scope="col">Status</th>
-                  {% endif %}
-                  <th class="govuk-table__header govuk-!-text-align-right" scope="col">Days</th>
-                </tr>
-                {% for charge in ada.charges %}
-                  <tr>
-                    <td class="govuk-table__cell">
-                      {{ charge.chargeNumber }}
-                    </td>
                     <td class="govuk-table__cell">
-                      {{ charge.toBeServed }}
+                      {% if not params.checkboxes %}
+                        <span class="moj-badge moj-badge--blue">{{ ada.status }}</span>
+                      {% endif %}
+                      {% if charge.status === 'PROSPECTIVE' %}
+                        <span class="moj-badge moj-badge--purple">{{ charge.status }}</span>
+                      {% endif %}
                     </td>
-                    <td class="govuk-table__cell">
-                      {{ charge.heardAt }}
-                    </td>
-                    {% if not params.hideStatuses %}
-                      <td class="govuk-table__cell">
-                        {% if not params.checkboxes %}
-                          <span class="moj-badge moj-badge--blue">{{ ada.status }}</span>
-                        {% endif %}
-                        {% if charge.status === 'PROSPECTIVE' %}
-                          <span class="moj-badge moj-badge--purple">{{ charge.status }}</span>
-                        {% endif %}
-                      </td>
-                    {% endif %}
-                    <!-- Days -->
-                    <td class="govuk-table__cell govuk-!-text-align-right">{{ charge.days }}</td>
-                  </tr>
-                {% endfor %}
-              </table>
-            </td>
-          </tr>
-        {% endfor %}
+                  {% endif %}
+                  <!-- Days -->
+                  <td class="govuk-table__cell govuk-!-text-align-right">{{ charge.days }}</td>
+                </tr>
+              {% endfor %}
+              <tfoot>
+              <tr class="govuk-table__row">
+                <th class="govuk-table__cell govuk-!-text-align-left" scope="row" colspan="3">Total</th>
+                <td class="govuk-table__cell govuk-!-text-align-right">{{ ada.total }}</td>
+              </tr>
+              </tfoot>
+            </table>
```

Comment:
lines 90-95 have been added... the rest are just formatting changes. Will speak to the team so we all have the same formatting config